ingress/helm-agic-config-template.txt

```diff
@@ -27,8 +27,8 @@ appgw:
 # Leaving this variable out or setting it to blank or empty string would
 # result in Ingress Controller observing all acessible namespaces.
 #
-kubernetes:
-    watchNamespace: ${namespaces}
+#kubernetes:
+#    watchNamespace: ${namespaces}
```

Comment:
Is the plan to reinstate these commented-out lines at some point?

Reply:
Nope, I'll properly comment them and remove the namespace sed functionality from azure-pipelines. Have left the default template intact (but hashed out) as per AGIC install recommendations.